src/phoneField/index.tsx

```diff
@@ -180,39 +194,44 @@ export default class PhoneField extends PureComponent<PhoneFieldProps, PhoneFiel
       textFieldPlaceholder,
       defaultPhoneValue,
       isInline,
+      error,
     } = this.props
 
     const baseClassName = cc([prefix({ phoneField: true })])
+    const wrapperClassName = `${baseClassName}-wrapper`
 
     const classNames = cc([
-      isInline ? `${baseClassName}--inline` : baseClassName,
+      isInline ? `${wrapperClassName}--inline` : wrapperClassName,
       this.props.className,
-      this.state.hasFocus && `${baseClassName}--hasFocus`,
+      this.state.hasFocus && `${wrapperClassName}--hasFocus`,
     ])
 
     return (
-      <div id={id} className={classNames} aria-labelledby={ariaLabelledBy}>
-        <SelectField
-          name={FIELDS.PHONEREGION}
-          options={this.state.countryData}
-          ariaLabel={selectFieldLabel}
-          defaultValue={this.props.defaultRegionValue}
-          onChange={this.handleChange}
-          onFocus={this.onFocus}
-          onBlur={this.onBlur}
-          focusBorder={!isInline}
-        />
-        <TextField
-          type="tel"
-          name={FIELDS.PHONENUMBER}
-          placeholder={textFieldPlaceholder}
-          defaultValue={defaultPhoneValue}
-          onChange={this.handleChange}
-          title={textFieldTitle}
-          onFocus={this.onFocus}
-          onBlur={this.onBlur}
-          focusBorder={!isInline}
-        />
+      <div className={cc([baseClassName, prefix({ error: !!error })])}>
+        <div id={id} className={classNames} aria-labelledby={ariaLabelledBy}>
+          <SelectField
+            name={FIELDS.PHONEREGION}
+            options={this.state.countryData}
+            ariaLabel={selectFieldLabel}
+            defaultValue={this.props.defaultRegionValue}
+            onChange={this.handleChange}
+            onFocus={this.onFocus}
+            onBlur={this.onBlur}
+            focusBorder={!isInline}
+          />
+          <TextField
```

Comment:
IF we want to do a11y properly, we will need a aria-invalid="true" on the textfield input
It might be out of scope for this PR but having a TODO/issue would be nice.

Reply:
So I created a dedicated ticket to check/fix A11y
>> BBCSPA-3355